### ISSUE TITLE
Set Object Mass

### ISSUE THREAD

**Issue by GodPhase777** [studio-minus/ppg-bugs-and-requests#1035]
**Feature description**
Add a set mass button.

**Feature purpose**
Weights other than predefined ones could be used. Useful for blocks that won't stop moving when something heavier is placed on them.

**Comment by DindinYT37** [comment]
Makes sense but I think weight multipliers should also be an option as it is more "newbie"-friendly.

**Comment by GodPhase777** [comment]
Weight multipliers might be good too, but to have both of those things would be too much i think, the mass setting is better then multiplier for people more experienced as it is more precise but multiplier is a better thing for newer people.
Maybe have them both be in the same menu? a multiply option and a specific mass option? That could open a door for a mass multiplier ray that makes things x times as heavy or x times as light.

**Comment by Drogo-man** [comment]
This sounds really epic tbh
